src/index.html

```diff
@@ -6,13 +7,69 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/styles.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__content">
+        <div class="card__image"></div>
+
+        <div class="card__title">
+          <p class="card__title-text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+        </div>
+
+        <div class="card__code">
+          <p class="card__code-text">Product code: 195434</p>
+        </div>
+
+        <div class="card__rating">
+          <div class="card__rating-stars">
+            <div class="card__rating-star"></div>
+            <div class="card__rating-star"></div>
+            <div class="card__rating-star"></div>
+            <div class="card__rating-star"></div>
+            <div class="card__rating-star"></div>
+          </div>
+          <p class="card__rating-text">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <div class="card__buy">
+          <a
```

Comment:
The 'Buy' button should be a `button` element or an `a` element with a valid `href` attribute pointing to the purchase page or action, not just a `#` placeholder.